pages/curriculum.tsx

```diff
@@ -171,7 +171,12 @@ export const Curriculum: React.FC<Props> = ({ lessons, alerts }) => {
   })
   return (
     <>
-      <ConnectToDiscordModal show={showConnectToDiscordModal} />
+      <ConnectToDiscordModal
+        show={showConnectToDiscordModal}
+        close={
+          /* istanbul ignore next */ () => setShowConnectToDiscordModal(false)
```

Comment:
Hmm so you are saying that the this line is actually covered by a test, but code cov doesn't recognize that its covered?

Reply:
I don't see a test in here that corresponds to to adding this `close` button update.
https://github.com/garageScript/c0d3-app/blob/master/__tests__/pages/curriculum.test.js

Maybe if you updated the test, the line would be covered.

Reply:
any idea how to test this close prop? this modal is kind of a child component of <Curriculum />
i already test the ConnectToDiscordModal in its own test file.

Reply:
I think what's not being tested is the calling of the function inside of the function you are passing to `close` as a prop. 

```
setShowConnectToDiscordModal(false)
```

Reply:
I think what's not being tested is the calling of the function inside of the function you are passing to close as a prop.

Reply:
That's what it looks like to me too. I figured if it's a function that calls a useState method, there's not much to test.

Reply:
Finally figured out how to cover it!